src/plugins/_api/serverList.ts

```diff
@@ -32,9 +32,9 @@ export default definePlugin({
             }
         },
         {
-            find: "#{intl::SERVERS}),children",
+            find: '#{intl::SERVERS}),gap:"xs",children',
```

Comment:
we can use something like this instead
```suggestion
            find: '.setGuildsTree(',
```